### ISSUE TITLE
Implementation of comparative experiments 

### ISSUE THREAD

**Issue by smlouva** [csmliu/STGAN#52]
When you reproduced Stargan, did you use the pytorch version of the code or the tensorflow version of the code? At the same time, should the labels input during training be represented by -0.5 and 0.5 like STGAN? thank you for your reply .

**Comment by CastellanLiu** [comment]
I used the PyTorch version, and the way to preprocess the labels were not changed (i.e., the same with StarGAN). Unfortunately, the portable disk was broken after I left Baidu, so the code and models are unavailable now.

**Comment by smlouva** [comment]
> I used the PyTorch version, and the way to preprocess the labels were not changed (i.e., the same with StarGAN). Unfortunately, the portable disk was broken after I left Baidu, so the code and models are unavailable now.

Thank you so much!
